lms/djangoapps/teams/static/teams/js/collections/team.js

```diff
@@ -3,6 +3,8 @@
     define(['teams/js/collections/base', 'teams/js/models/team', 'gettext'],
         function(BaseCollection, TeamModel, gettext) {
             var TeamCollection = BaseCollection.extend({
+                sortField: 'name',
```

Comment:
As a philosophical question, I'm not sure how to handle defaults like this. The server API has a default sort order already, so I think it would be better to have the default be null, and then have the parse method pick it out. Does the API return the sort order that was used? 

I see this issue with the page sizes too. We have default sizes sprinkled around both the front end and the API. I don't know whether it makes sense to have the front end just pick up the server's default, or whether it is clearer for the front-end to have its own declared defaults.

You don't have to do anything about this, but it is something I've been wondering about for a few days now.